Heimdallr/Core/Heimdallr.swift

```diff
@@ -208,6 +219,7 @@ public let HeimdallrErrorNotAuthorized = 2
                             }
                             return .Failure(error)
                         }))
+                        self.releaseRefreshQueue()
```

Comment:
I thought about that and since it can only be used in this completion block I left it out. But you are right, it's still a bit nicer. Fixed!